lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -430,12 +430,18 @@
 
 
       <?php // Custom Share Modal // ?>
-      <?php if ($custom_social_share_link) : ?>
+      <?php if ($custom_social_share_link && $register_voters) : ?>
         <div data-modal id="modal-share" role="dialog">
           <div class="modal__block">
-            <h3><?php print t('Share this Campaign') ?></h3><br>
+            <h3><?php print t('Share This Now!') ?></h3><br>
```

Comment:
ugh I know this is annoying, but ideally we want to re-use this custom link across the site, and don't want it tied to votes.

Can we pre-process the text here and print it in here if `register_voters` is set? instead of requiring that var to display the custom share?